recipes/platform.hashing/all/conanfile.py

```diff
@@ -47,10 +47,6 @@ def _compilers_minimum_version(self):
     def _minimum_cpp_standard(self):
         return 20
 
-    def requirements(self):
-        if Version(self.version) >= "0.3.0":
-            self.requires("platform.delegates/0.2.7")
```

Comment:
Why is this removed? 🤔 it was a added because of a bug AFAIK

Reply:
one of our developers suggested that this package would be useful in development, but in the end it was not used when the new version was released